File: LinearSearchAlgo/Linearsearch.java
import java.util.Scanner;

public class Linearsearch {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }

        int s = sc.nextInt();

        for (int i = 0; i < n; i++) {
            if (arr[i] == s) {
                System.out.println(s + " Found at index " + i);
                return;
            }
        }
        System.out.println("Not Found");
    }
}

File: LinearSearchAlgo/Linear_search.c
#include<stdio.h>
int main(){
    int n,s,i;
    scanf("%d",&n);
    int arr[n];
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    scanf("%d",&s);
    for(i=0;i<n;i++){
        if(s==arr[i]){
            printf("%d Found at index %d",s,i);
            return 0;
        }
    }
    printf("Not Found");
    return 0;
}